controllers/addon/operator_controller_test.go

```diff
@@ -0,0 +1,169 @@
+package addon
+
+import (
+	"context"
+	"github.com/go-logr/logr"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I'd prefer to give it a name to `metav1`.

Reply:
good catch!